avm/res/machine-learning-services/workspace/main.bicep

```diff
@@ -547,6 +562,87 @@ type featureStoreSettingType = {
   onlineStoreConnectionName: string?
 }?
 
+@discriminator('type')
+type OutboundRuleType = FqdnOutboundRuleType | PrivateEndpointOutboundRule | ServiceTagOutboundRule
+
+type FqdnOutboundRuleType = {
+  @sys.description('Required. Type of a managed network Outbound Rule of a machine learning workspace. Only supported when \'isolationMode\' is \'AllowOnlyApprovedOutbound\'.')
+  type: 'FQDN'
+
+  @sys.description('Required. Fully Qualified Domain Name to allow for outbound traffic.')
+  destination: string
+
+  @sys.description('')
```

Comment:
Missing description. Something along the lines of 'Optional. ...'

Reply:
Apologies, not sure how I missed that